packages/preset-mini/src/_rules/position.ts

```diff
@@ -1,9 +1,26 @@
 import type { CSSEntries, Rule, RuleContext, StaticRule } from '@unocss/core'
 import type { Theme } from '../theme'
-import { globalKeywords, h, insetMap, makeGlobalStaticRules } from '../utils'
+import { globalKeywords, h, insetMap, makeGlobalStaticRules, makeGroupAutocomplete } from '../utils'
+
+const positionValues = [
+  'relative',
+  'absolute',
+  'fixed',
+  'sticky',
+  'static',
+  ...globalKeywords,
+]
 
 export const positions: Rule[] = [
-  [/^(?:position-|pos-)?(relative|absolute|fixed|sticky)$/, ([, v]) => ({ position: v })],
+  [/^(?:position-|pos-)?(relative|absolute|fixed|sticky)$/, ([, v]) => ({ position: v }), {
+    autocomplete: [
+      makeGroupAutocomplete([
+        ['position', 'pos'],
+        positionValues,
+      ]),
+      makeGroupAutocomplete([positionValues]),
```

Comment:
I'd actually prefer to have plain string there. For the position values, we could put it in https://github.com/unocss/unocss/blob/995cc6270c167537c7ab70c94696022e5220e813/packages/autocomplete/src/parse.ts#L6

Reply:
Now we can add a custom shorthands in preset-mini